tests/assets/sub0/misc/cato_structural_config.json

```diff
@@ -0,0 +1,56 @@
+{
+   "general": {
+      "fslRootDir": "/home/work/local/fsl/",
+      "freesurferRootDir": "/usr/local/freesurfer/7.2.0",
+      "templates": [
+         "phantomatlas"
+      ],
+      "reconstructionMethods": [
+         "DTI",
+         "GQI",
+         "GQI_DTI"
+      ],
+      "reconstructionSteps": [
+         "reconstruction_diffusion",
+         "reconstruction_fibers",
+         "reconstruction_fiber_properties",
+         "reconstruction_network"
+      ],
+      "ROIsFile": "misc/ROIs_TEMPLATE.txt",
+      "outputDir": "dwi_processed"
+   },
+   "parcellation": {
+      "forceFreesurferOverwrite": false
+   },
+   "reconstruction_diffusion": {
+      "exportNifti": {
+         "exportNifti": true,
+         "measures": [
+            "fractional anisotropy",
+            "axial diffusivity",
+            "radial diffusivity",
+            "mean diffusivity"
+         ]
+      }
+   },
+   "reconstruction_fibers": {
+      "NumberOfSeedsPerVoxel": 8,
```

Comment:
Currently these tests take about ~5 min on my local machine. That's okay, but maybe it's better to set this to 1

Reply:
Yes, that seems ok for now. But might have to be adjusted when the number of test cases increases in the future.